lib/Slackbot_worker.js

```diff
@@ -358,6 +358,28 @@ module.exports = function(botkit, config) {
         bot.say(msg,cb);
     };
 
+    bot.startTyping = function(src) {
```

Comment:
We started using Javascript Documentation found in http://usejsdoc.org.

Can you update all of your functions with necessary documentations?